README.md

```diff
@@ -80,7 +80,7 @@ If you want the package detection to work, make sure you attach packages to the
  * macOS package name should start with "osx-"
  * linux package name should start with "lin-"
 
-For a real life example, check a [release](https://github.com/Gibberlings3/Tweaks-Anthology/releases/latest) of Tweaks Anthology.
```

Comment:
come on, it was perfect and now you've introduced an error.

Reply:
done